libraries/DNSServer/src/DNSServer.h

```diff
@@ -1,12 +1,20 @@
 #ifndef DNSServer_h
 #define DNSServer_h
 #include <WiFiUdp.h>
+#include <memory>
```

Comment:
As I explained in my reply, this include shouldn't be necessary, because you can keep the signature for respondToRequest as it was before.

Reply:
Removed it